soc/arm/nuvoton_npcx/common/soc_rom_hdr.c

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2020 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * Header for Nuvoton NPCX EC.
+ *
+ * This header is used by Nuvoton EC Booter (ROM Code).
+ */
+
+#include <stdint.h>
+#include <kernel.h>
+
+/* Signature used by fw header */
+#define SIG_FW_EC           0x2A3B4D5E
```

Comment:
How about using lower-case hex? It is easier to read

Reply:
Done.